dictionary/get_vdem.R

```diff
@@ -1,12 +1,17 @@
 source(here::here('dictionary/utilities.R'))
 
-vdem <- readRDS('dictionary/data_raw/data_vdem_v8_april2018.rds') %>%
+# readRDS("~/countrycode/dictionary/data_raw/V-Dem-CY-Core-v12.rds") %>%
+#   arrange(country_text_id,year) %>%
+#   select(country_name,country_id,year) %>%
+#   saveRDS(vdem, file = "~/countrycode/dictionary/data_raw/data_vdem_v12_march2022.rds", version = 2, compress = "xz")
```

Comment:
maybe get rid of this commented code so it doesn't confuse us later?

Reply:
I included it deliberately. `data_vdem_v8_april2018.rds` appears to have been a reduced version of the V-Dem dataset. This code reminds us how to munge the V-Dem dataset into the reduced version. For example, it took me a little while to understand that the dataset was sorted on a variable that was no longer present in the reduced dataset.